pkg/webui/console/containers/gateways-table/index.js

```diff
@@ -85,6 +86,7 @@ export default class GatewaysTable extends React.Component {
       totalCount: selectGatewaysTotalCount(state),
       fetching: selectGatewaysFetching(state),
       error: selectGatewaysError(state),
+      mayAdd: checkFromState(mayCreateGateways, state),
```

Comment:
In that case the table row will show the 'unknown' status, based on the 403 response that it got from the stat endpoint. I think that is good enough. We'd otherwise have to additionally query the rights list for every gateway in the list.